index.md

```diff
@@ -742,6 +750,11 @@ for the version control system, even if the data itself has not changed.
 > data in public version control systems. Some institutions may provide
 > access to private version control systems, so it is worth checking
 > with your IT department.
+>
+> Another type of data that may be shared unintentionally is security
+> credentials such as passwords and private keys. Remove files with
+> sensitive data from version control and when possible store this
+> information in environment variables.
```

Comment:
This is a great point (I am struggling with that right now) but I don't know if the audience will know what `environment variables` means.